new/LIT7048Qene.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7048Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ከልአኪ፡ አሮን፡ በበትረ፡ እዴሁ፡ ጥልልት።</title>
```

Comment:
Indeed, to Jonas idea, I see that ከልአኪ፡ አሮን፡ በበትረ፡ እዴሁ፡ ጥልልት። ወሕዝቅኤል፡ ጽባሐዊ፡ ዓቀመኪ፡ በኆኅት። was used in another qene.

Reply:
Do we keep this text as one qene poem nevertheless, or should we split into two? Should I refer to the other qene in a relation and in the abstract?

Reply:
We keep it as one. A reference to other qene in abstract is ok, we will see if we can collect more cases like this (I expect we will have some), to see if it can be encoded. It is indeed curious if the first two verses, having been borrowed, keep the rhyme of the source. What would qene masters say about this? (the rule is that qenes should be fully original, but the reality is a different thing, there are treatise offering ready verses for qenes) :)